index.js

```diff
@@ -1,32 +1,25 @@
+/* jshint node: true */
 'use strict';
 
-var path = require('path');
-var fs = require('fs');
 var autoprefixer = require('broccoli-autoprefixer');
-var defaults = require('lodash').defaults;
+var defaults     = require('lodash').defaults;
 
-function EmberCLIAutoprefixer(project) {
-  this.project = project;
-  this.name = 'Ember CLI Autoprefixer';
-}
+module.exports = {
+  name: 'ember-cli-autoprefixer',
+  included: function(app, parentAddon) {
+    this.app = app;
+    this.options = defaults(this.app.options.autoprefixer || {}, {
+      enabled: true
+    });
+    this.enabled = this.options.enabled;
+  },
+  postprocessTree: function(type, tree) {
+    var tree;
 
-EmberCLIAutoprefixer.prototype.postprocessTree = function (type, tree) {
-  if ((type === 'all' || type === 'styles') && this.enabled) {
-    tree = autoprefixer(tree, this.options);
-  }
-
-  return tree;
-};
+    if ((type === 'all' || type === 'styles') && this.enabled) {
+      tree = autoprefixer(tree, this.options);
+    }
 
-EmberCLIAutoprefixer.prototype.included = function included(app) {
-  this.app = app;
-  this.options = defaults(this.app.options.autoprefixer || {}, {
-    enabled: true
-  });
-  this.enabled = this.options.enabled;
-  delete this.options.enabled;
```

Comment:
I delete it because `this.options` is passed on directly as options for `broccoli-autoprefixer`, and I didn't want to pass on this extra option, just in case.